src/services/tasks.service.ts

```diff
@@ -1,4 +1,5 @@
 import { Task } from '../models/task.model';
+import { ETaskStatus } from '../models/share';
 
 export const getAllTasks = async (): Promise<Task[]> => {
     return await Task.findAll();
```

Comment:
No need await here. As we are returning a promise, the promise itself can flow out all the way to the upper level to be fullfilled

Reply:
We only use await here if we want to do something with the list of Tasks from the output of `Task.findAll()`